OneCommunityGlobal/HighestGoodNetworkApp
# Jiayu add teams limit see all

# Description
<img width="707" alt="image" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/58479595/f8a0d795-c931-4d68-9f97-64e004708ca4">

## Related PRS:
This frontend PR is based on PR #1905 
To test this frontend PR you need to checkout [backend PR 1017](https://github.com/OneCommunityGlobal/HGNRest/pull/1017).

## Main changes explained:
<img width="472" alt="image" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/58479595/74b84cfc-ee34-4654-9f41-ae2b71f0b98b">

- Added a loading state for team members between clicking different teams


## How to test:
1. check into current branch
2. do `npm install` and `npm run start:local` to run this PR locally
3.  log as admin user
4.  Go to Other Links ->Teams
5.  Create two Teams , lets say Team A and Team B (you can use any existing Team as well)
6.  Add  the a **_testuser_1_** as team member to **_Team A_** **_and Team B_**
7.  Set the toggle switch to off  in Team A for **_testuser_1_**
8.  Login to the **_testuser_1_**'s  profile and verify if the user is only able  to see  the members of the  Team B and not Team A

## Screenshots or videos of changes:
https://www.loom.com/share/80d08159a46f40c8ab9871d27d729044?sid=1537e57a-a323-4124-84e8-37773be5d1b9
